test_unstructured/chunking/test_title.py

```diff
@@ -189,7 +190,7 @@ def test_split_elements_by_title_and_table():
         Title("A Great Day"),
         Text("Today is a great day."),
         Text("It is sunny outside."),
-        Table("<table></table>"),
+        Table("Heading\nCell text"),
```

Comment:
This global change is purely psychological, but `"<table></table>" would never appear as `Table.text`, rather it would appear as `Table.metadata.text_as_html`. This apparent misconception could be related to the underlying mistake in the code leading to this so I made these more realistic so the reader's intuition about these elements is not led astray.